Update inf-addservice-directive.md (#3517)
MicrosoftDocs/windows-driver-docs: windows-driver-docs-pr/install/inf-addservice-directive.md

**0x00020000** (SPSVCINST_NOCLOBBER_FAILUREACTIONS) (Windows 11 version 22H2 and later versions of Windows)  
Do not overwrite the failure actions for the given service if this service already exists in the system and has existing failure actions. See the FailureActions directive below for more information on failure actions.

> **0x00040000** (SPSVCINST_NOCLOBBER_BOOTFLAGS) (Windows 11 build 25381  and later versions of Windows)

Reviewer: There is a double space after "25381".
```suggestion
**0x00040000** (SPSVCINST_NOCLOBBER_BOOTFLAGS) (Windows 11 build 25381 and later versions of Windows)
```

Reviewer 2: Fixed

> Do not overwrite the boot flags value for the given service if this service already exists in the system and has an existing value. See the BootFlags directive below for more information

Reviewer: missing period at the end of the second sentence

[AddReg=add-registry-section[, add-registry-section] ...]
[DelReg=del-registry-section[, del-registry-section] ...]
[BitReg=bit-registry-section[,bit-registry-section] ...]

> [BootFlags=value] (Windows 11 build Version <TBD> and later versions of Windows) 

Reviewer: move this to the end since it is the newest (makes the parentheticals about what OS versions it is supported in go in order)

**BitReg**=*bit-registry-section*[**,**_bit-registry-section_]...  
Is valid in a *service-install-section* but almost never used.
 

> **BootFlags**=*value*

Reviewer: move all of the BootFlags docs after the AddTrigger docs below

> **BootFlags**=*value*

Reviewer: The section "Promoting a Driver's StartType at Boot Depending on Boot Scenario" later on this page should be updated to indicate that before Windows 11 version <TBD>, you should add BootFlags with the AddReg as it specifies but after Windows 11 version <TBD>, you should use this new BootFlags directive.

> [!NOTE]

> > This value can only be used with kernel-mode driver services and is only available with Windows 11 Build 25371 and later. 

Reviewer: ```suggestion
> This value can only be used with kernel-mode driver services and is only available with Windows 11 Build 25381 and later. 
```

HKR,,BootFlags,0x00010003,0x14 ; CM_SERVICE_USB3_DISK_BOOT_LOAD|CM_SERVICE_USB_DISK_BOOT_LOAD
```

> [!NOTE]

> > This syntax should be used only in Windows 11, builds older than 25381, for all others use **BootFlags** directive.

Reviewer: I think the current text might imply that you can only use the AddReg in Windows 11 builds, but builds older than 25381.  The AddReg is valid in Win10 and below also, so suggesting a potential re-wording to make that clearer.
```suggestion
> This syntax should be used only in Windows versions and builds older than Windows 11 build 25381; for Windows 11 build 25381 and later, use the **BootFlags** directive.
```

Reviewer 2: Fixed
